study-builder/studies/testboston/subs.conf

```diff
@@ -114,6 +120,7 @@
       "household_diagnosed": { include required("snippets/question-household-diagnosed.conf") },
       "contact_diagnosed": { include required("snippets/question-contact-diagnosed.conf") },
       "interested_continuing": { include required("snippets/question-interested-continuing.conf") },
+      "flu_shot": { include required("snippets/question-flu-shot.conf") },
```

Comment:
This question is only in the longitudinal followup survey right? Why is it included under `covid_survey`?

Reply:
I guess I didn't want to add followup_survey entry for just this question. will change it.